src/test/scala/org/broadinstitute/dsde/firecloud/service/LibraryApiServiceSpec.scala

```diff
@@ -204,6 +224,15 @@ class LibraryApiServiceSpec extends BaseServiceSpec with LibraryApiService {
           }
         }
       }
+      "GET on " + libraryGroupsPath - {
+        "should return the all broad users group" in {
+          new RequestBuilder(HttpMethods.GET)(libraryGroupsPath) ~> dummyUserIdHeaders("1234") ~> sealRoute(libraryRoutes) ~> check {
+            status should equal(OK)
+            val respdata = response.entity.asString.parseJson.convertTo[Seq[String]]
+            assert(respdata.forall { g => FireCloudConfig.ElasticSearch.discoverGroupNames.contains(g) })
```

Comment:
greg, your response was for a different part of the code, and i made the change there. this is the test and these groups should always have the same members